### PR TITLE
feat: added pc updates from bugs list

### PR DESCRIPTION
**Pull request by eugenederii**

### CODE REVIEW

File: src/assets/images/profile1uk.png
Comment (VadimTarasenko) [inline src/assets/images/profile1uk.png:1]: Rename user to "Emma Bloom"

File: src/modules/booking/components/booking-preview-progressbar/booking-preview-progressbar.tsx [hunk 1]
```diff
@@ -35,23 +35,23 @@ export const BookingPreviewProgressbar: FC<BookingPreviewProgressbarProps> = ({
...
         })}
       />
       <div className='absolute bottom-0 left-0 right-0 top-0 flex flex-col items-center justify-center'>
         <span className='text-sm text-gray-accent'>
-          {intl.formatMessage({
-            id: 'progressbar.title',
-          })}
+          {daysRemaining > 0
+            ? intl.formatMessage({ id: 'progressbar.title.lost' })
+            : intl.formatMessage({ id: 'progressbar.finished' })}
```
Comment (VadimTarasenko) [inline src/modules/booking/components/booking-preview-progressbar/booking-preview-progressbar.tsx:54]: We also need to handle cases where only hours or minutes remain. For example, if 30 minutes are left for the event, we should display '30 minutes left'

File: src/modules/booking/containers/booking-preview/booking-preview.tsx [hunk 2]
```diff
@@ -111,9 +87,9 @@ export const BookingPreview = () => {
   return (
     <main className='mx-auto mb-20 mt-20 flex w-full max-w-[950px] flex-1 flex-col px-[15px]'>
       {/* todo : booking number */}
-      <Typography className='mx-auto text-center !text-[24px]'>
+      {/* <Typography className='mx-auto text-center !text-[24px]'>
         {intl.formatMessage({ id: 'booking.number' })} № 20
-      </Typography>
+      </Typography> */}
```
Comment (VadimTarasenko) [inline src/modules/booking/containers/booking-preview/booking-preview.tsx:92]: we can remove it

**Review by IhorBalan — APPROVED** [review]
Fix issue that Vadym described and you can merge

**Review by VadimTarasenko — APPROVED** [review]

**Pull request merged**